SPEC.md

```diff
@@ -245,7 +245,15 @@ For example, if the ACE is looking for `example.com/reduce-worker` it will reque
 
     https://example.com/reduce-worker?ac-discovery=1
 
-Then inspect the HTML returned for meta tags that have the following format:
+As a convenience feature, trying to fetch an app container image with a port number, will assume that the discovery server is listening in this port.
```

Comment:
I don't want to add it to this part of the spec. I think if anything it should fall under #4 in a suggested format for convenience etc; then we can reference that from here.